modules/rosa-hcp-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,204 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-hcp-shared-vpc-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sharing-vpc-creation-and-sharing_{context}"]
+= Step One - VPC Owner: Configuring a VPC to share within your AWS organization
+
+You can share subnets within a configured VPC with another AWS user account if that account is within your current AWS organization.
+
+image::522-shared-vpc-step-1.png[]
+.Procedure
+
+. Create or modify a VPC to your specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console].
++
+. Create the `Route 53 role`.
+.. Create a custom policy file to allow for necessary shared VPC permissions that uses the name `SharedVPCPolicy`:
++
+[source,terminal]
+----
+$ cat <<EOF > /tmp/shared-vpc-policy.json
```

Comment:
Policies need to be changed to reflect the [following](https://github.com/openshift/managed-cluster-config/blob/e6dc8f2d258883f080ce1fada4f9912e856d9f6d/resources/sts/4.17/hypershift/openshift_hcp_shared_vpc_route_53_credentials_policy.json)